its/src/test/java/com/sonar/it/scanner/msbuild/utils/TestUtils.java

```diff
@@ -150,20 +150,20 @@ public static Location getMavenLocation(String groupId, String artifactId, Strin
   }
 
   public static void createVirtualDrive(String drive, Path projectDir, String subDirectory) {
-    int setupStatus = CommandExecutor.create().execute(
-      Command.create("SUBST")
-        .addArguments(drive)
-        .addArguments(projectDir.resolve(subDirectory).toAbsolutePath().toString())
-        .setDirectory(projectDir.toFile()),
-      TIMEOUT_LIMIT);
-    assertThat(setupStatus).isZero();
+    var target = projectDir.resolve(subDirectory).toAbsolutePath().toString();
+    var result = GeneralCommand.create("SUBST", projectDir)
+      .addArgument(drive)
+      .addArgument(target)
+      .execute();
+    assertThat(result.isSuccess()).withFailMessage("SUBST create failed. Most likely flakiness from a previous canceled run that did not clean-up the drive. " +
+      "The mapping of drive %s to folder '%s' failed with logs:%n%s", drive, target, result.getLogs()).isTrue();
```

Comment:
This line will never fail now, because of the `assertThat(result.isSuccess())` in GeneralCommand.execute. Do we need the extended logging from here?